src/types.js

```diff
@@ -59,9 +63,70 @@ export type SpacingAliases = {
   +[spacingAlias: string]: number,
 }
 
+export type DisplayAliases = {
+  +[displayAlias: string]: string,
+}
+
 export type ConfigProviderContext = {|
   +xnReflex: {|
+    +displayAliases: DisplayAliases,
     +spacingAliases: SpacingAliases,
     +base: number,
   |},
 |}
+
+export type OneResolutionGrid = {
+  align?: AlignGrid,
+  children?: React.Node,
+  className?: string,
+  dev?: Dev,
+  justify?: Justify,
+  margin?: Spacing,
+  marginBottom?: SpacingValues,
+  marginLeft?: SpacingValues,
+  marginRight?: SpacingValues,
+  marginTop?: SpacingValues,
+  padding?: Spacing,
+  paddingBottom?: SpacingValues,
+  paddingLeft?: SpacingValues,
+  paddingRight?: SpacingValues,
+  paddingTop?: SpacingValues,
+  show?: DisplayValues,
+  size?: Size,
+  style?: { [string]: string | number },
+}
+
+export const resolutionProperties = [
```

Comment:
This is conflating flow types, and properties used by reflex. They are different thing in my mind, and maybe shouldn't be grouped together in the same module.

Reply:
that's fair, I was thinking about this, imo it made sense to put them here so that there's a single spot to look at when new properties are added. Maybe the right solution would be to generate this array but that also feels overkill

Reply:
I also find this confusing.